package.json

```diff
@@ -26,7 +26,7 @@
   "dependencies": {
     "passport-strategy": "*",
     "xml2js": "~0.4.1",
-    "xml-crypto": "0.0.22",
+    "xml-crypto": "latest",
```

Comment:
This should be locked to a minor version.  `Latest` is usually not a good choice in the node ecosystem as it can introduce breaking changes.